lib/openapi_parser/schemas/security_schemes.rb

```diff
@@ -0,0 +1,13 @@
+module OpenAPIParser::Schemas
+  class SecurityScheme < Base
+    # @!attribute [r] jwt
+    #   @return [Jwt, nil]
+    openapi_attr_object :jwt, Jwt, reference: false
```

Comment:
First, the library defines a class with the same data structure as the OpenAPI Specification.
Therefore, SecurityScheme should have a `type` and `scheme`...
(Since the class Jwt is not in the OpenAPI definition).

Reply:
the `openapi_attr_object` can be any name. How can we allow any arbitrary name?

I am thinking of this after getting `securitySchemes` string hash:

```
    scheme.each_with_index do |k, v|
      if k[v]["type"] = "http" && k[v]["scheme"] = "bearer" && k[v]["bearerFormat"] = "JWT"
        # run validation code here
      end
    end
```

Reply:
The class `OpenAPIParser::Schemas::XXXX` has the same fields as the object of the same name defined in the OpenAPI Specification.
The `openapi_attr_xxxx` method is a helper function to define instance variables of the class.
Therefore, in this case, we can define fields such as type, description, scheme... and use those instance variables in validato.

```
    openapi_attr_values :type, :description, :schema......
    openapi_attr_value :bearer_format, schema_key: : bearerFormat
```

```
def validate_security_schemes(params, options)
  if self.type == "http" && self.schema == "bearer" && self.bearer_format == "JWT"
     # run validation code here
  end
end
```

Reply:
SecuritySchemas object is root can have any names below code needs to depend on a name (string)

example:

```yaml
components:
  securitySchemas:
    MyBearer: <--- This can be anything
      type: http
      scheme: bearer
      bearerFormat: JWT
``` 

```ruby
  if self.type == "http" && self.schema == "bearer" && self.bearer_format == "JWT"
     # run validation code here
  end
```

won't work from what i see.

Reply:
@ota42y This is a blocked. I cannot the `securityScheme` name.

```yaml
components:
  securitySchemes:
    MyBearer: <--- This can be anything
      type: http
      scheme: bearer
      bearerFormat: JWT
```